cel/cos_tlv_test.go

```diff
@@ -0,0 +1,62 @@
+package cel
+
+import (
+	"bytes"
+	"crypto"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-tpm-tools/client"
+	"github.com/google/go-tpm-tools/internal/test"
+	pb "github.com/google/go-tpm-tools/proto/attest"
+)
+
+func TestCosEventlog(t *testing.T) {
+	tpm := test.GetTPM(t)
+	defer client.CheckedClose(t, tpm)
+
+	hashAlgoList := []crypto.Hash{crypto.SHA256, crypto.SHA1, crypto.SHA512}
+	cel := &CEL{}
+
+	testEvents := []struct {
+		cosNestedEventType CosType
+		pcr                int
+		eventPayload       []byte
+	}{
+		{ImageRefType, 13, []byte("docker.io/bazel/experimental/test:latest")},
```

Comment:
Sorry missed this: use debug or application here and below as well.

Reply:
Done, I missed as well